libafl_qemu/src/emu.rs

```diff
@@ -395,6 +397,18 @@ pub fn remove_breakpoint(addr: u64) {
     }
 }
 
+pub fn set_hook(addr: u64, callback: extern fn()) {
```

Comment:
Is there a reason you're mixing insert and set in the API? Would be good to be consistent I think

Reply:
set_hook and set_breakpoint, it is consisent